servicecontrol/servicecontrol-in-practice.md

```diff
@@ -0,0 +1,35 @@
+- The [Auditing](/nservicebus/operations/auditing.md) plugin will capture a record of everything that passes through an endpoint. It is worth considering turning audit off in some or all endpoints in production or directing the audit traffic to a null queue. If production audits are needed consider directing the traffic to a well resourced dedicated server. Audits in ServiceControl are primarily for performing system analysis using ServiceInsight. If ServiceInsight is not deployed in production then avoid using audits in production.
+- [Heartbeats](/servicepulse/intro-endpoints-heartbeats.md) and [Custom Checks](/servicepulse/intro-endpoints-custom-checks.md) can enhance a system's monitoring capability, but they add extra noise. Often, not all endpoints are mission critical. Therefore it is worth making sure that the risks to the system if an endpoint is not available are examined and understood. All communication from endpoints with ServiceControl is performed via messaging. Adding a message to the queue every second may have little impact on when the notification shows up in ServicePulse. If ServiceControl is shutdown without pausing system Endpoints, heartbeats will continue to be added to the queue. ServiceControl will attempt to process all the messages in the queue when it restarts.
+
+##### Less can be more
+
+Moving forward, take a thoughtful approach in the adoption of plugins and features:
+
+- Read the [Capacity Planning Guide](/servicecontrol/capacity-and-planning.md) for ServiceControl.
+- Read the [Troubleshooting Guide](/servicecontrol/troubleshooting.md) before deployment in production.
+- [Turn off auditing](/nservicebus/operations/auditing.md) on all endpoints as well as [heartbeats and custom checks](/servicepulse/how-to-configure-endpoints-for-monitoring.md).
+- Perform load tests to baseline the solution.
+- When comfortable with the performance of the system try adding [Heartbeats](/servicepulse/intro-endpoints-heartbeats.md) to monitor the system again.
+- Try increasing the [heartbeat interval](/servicecontrol/plugins/heartbeat.md). Ideally heartbeat updates should not occur more frequently than ServiceControl can process them or more than Operations staff are prepared to monitor them with ServicePulse.
+- With each additional change perform a load test again adjusting the heartbeat interval satisfied with the result.
+- Repeat the process of considering the business relevance, system impact and load testing with each [Custom Check](/servicecontrol/plugins/custom-checks.md).
```

Comment:
I somewhat agree. We are in control of the message on the wire but: 

1. a user can supply a massive `reason` for failure
2. a user can specify an overly aggressive schedule for a Custom Check (i.e. check memory usage every 3 sec)
3. a user can have a very large number of Custom Checks

Of those I think that 2 is the most likely.